tests/test_style/test_colors.py

```diff
@@ -205,6 +205,75 @@ def test_colormap_cmap(self):
             (0.8, 0.8, 0.8, 1.0)
         ]
 
+    def test_colormap_palette_mpl(self):
+        """
+        Assert that supplying a maptlotlib palette as colormap works
+        """
+        cmap = cm.get_cmap('nipy_spectral')
+        colors = resolve_colors(colormap=cmap)
+        assert colors == [
+            (0.0, 0.0, 0.0, 1.0),
+            (0.0, 0.0, 0.8667, 1.0),
+            (0.0, 0.6667, 0.5333, 1.0),
+            (0.0, 1.0, 0.0, 1.0),
+            (1.0, 0.6, 0.0, 1.0),
+            (0.8, 0.8, 0.8, 1.0)
+        ]
+
+    def test_colormap_palette_yb(self):
+        """
+        Assert that supplying a yellowbrick palette as colormap works
+        """
+        cmap = ColorPalette('neural_paint')
+        assert resolve_colors(colormap=cmap) == [
+            (0.08627450980392157, 0.44313725490196076, 0.5725490196078431),
+            (0.43137254901960786, 0.4588235294117647, 0.2823529411764706),
+            (0.7725490196078432, 0.6352941176470588, 0.6705882352941176),
+            (0.0, 0.8, 1.0),
+            (0.8705882352941177, 0.47058823529411764, 0.6823529411764706),
+            (1.0, 0.8, 0.6),
+            (0.23921568627450981, 0.24705882352941178, 0.25882352941176473),
+            (1.0, 1.0, 0.8)
+        ]
+
+    def test_colormap_cmap_with_colors(self):
+        """
+        Assert that supplying a maptlotlib.cm as colormap works
+        """
```

Comment:
Would you mind updating the docstring for this test so that it's a bit more specific and easier to differentiate from the `test_colormap_palette_mpl` (which currently has approximately the same docstring)? Perhaps

```python
    """
    Assert that colors overrides a mpl colormap if both are provided
    """
```